### ISSUE TITLE
Infinite loop on error

### ISSUE THREAD

**Issue by noraj** [ticarpi/jwt_tool#9]
```
$ python jwt_tool.py 'ew...Ry' /usr/share/wordlists/password/rockyou.txt
```

```
Token header values:                                                                                                                                                                                                                                                            
[+] alg = ES512                                                                                                                                                                                                                                                                 
[+] typ = JWT
```

```
#  6: Crack signature with supplied dictionary file    #
```

When I use option 6 on a ES512, I have the following error.

```
Algorithm is not HMAC-SHA - cannot test with this tool.                                                                                                                                                                                                                         
```

Which seems normal but the problem is that it loops over it indefinitely.

## Asciinema

I made a asciinema "video" of it.

[asciinema.zip](https://github.com/ticarpi/jwt_tool/files/3810274/asciinema.zip)

To play it: `asciinema play xxx.cast`

**Comment by ticarpi** [comment]
Ah, interesting! Good spot.
I'll fix that in the next update, which should be out in a few days.
Even better news is that the new update will also bring support for tampering, signing and verifying ESxxx tokens too :)

**Comment by ticarpi** [comment]
Now fixed in v1.3
Thanks for raising.
